adapters/relevantdigital/relevantdigital.go

```diff
@@ -0,0 +1,278 @@
+package relevantdigital
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/buger/jsonparser"
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/macros"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint *template.Template
+	name     string
+}
+
+const RELEVANT_DOMAIN = ".relevant-digital.com"
+const DEFAULT_TIMEOUT = 1000
+const DEFAULT_BUFFER_MS = 250
+const STORED_REQUEST_EXT = "{\"prebid\":{\"debug\":%t,\"storedrequest\":{\"id\":\"%s\"}},\"relevant\":{\"count\":%d,\"adapterType\":\"server\"}}"
+const STORED_IMP_EXT = "{\"prebid\":{\"storedrequest\":{\"id\":\"%s\"}}}"
```

Comment:
scope of these constants is limited to `relevantdigital` package. 

In golang any package var/const/function starting with upper case character becomes public. Therefore should make above constants private

```

const relevant_domain = ".relevant-digital.com"
const default_timeout = 1000
const default_buffer_ms = 250
const stored_request_ext = "{\"prebid\":{\"debug\":%t,\"storedrequest\":{\"id\":\"%s\"}},\"relevant\":{\"count\":%d,\"adapterType\":\"server\"}}"
const stored_imp_ext = "{\"prebid\":{\"storedrequest\":{\"id\":\"%s\"}}}"

```